modules/3d/src/rgbd/odometry_settings_impl.cpp

```diff
@@ -0,0 +1,346 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include "../precomp.hpp"
+#include "utils.hpp"
+
+namespace cv
+{
+
+const cv::Matx33f defaultCameraMatrix =
+{ /* fx, 0, cx*/ 0, 0, 0,
+  /* 0, fy, cy*/ 0, 0, 0,
+  /* 0,  0,  0*/ 0, 0, 0 };
+const std::vector<int> defaultIterCounts = { 7, 7, 7, 10 };
```

Comment:
avoid declaration of global objects - they slowdown library initialization.
Use class ctors / initialization on-demand instead.

Reply:
if I understood right, done